src/tests/ftest/util/file_count_test_base.py

```diff
@@ -109,3 +113,41 @@ def run_file_count(self):
                 errors = True
         if errors:
             self.fail("Test FAILED")
+
+    def run_ior_with_flags(self, processes, ppn, create_pool=True, create_cont=True,
+                           plugin_path=None, mount_dir=None):
+        """Execute ior with optional overrides for ior flags and object_class.
+
+        If specified the ior flags and ior daos object class parameters will
+        override the values read from the yaml file.
+
+        Args:
+            processes (int): Number of processes.
+            ppn (int): Number of processes per node.
+            create_pool (bool, optional): If it is true, create pool and
+                container else just run the ior. Defaults to True.
+            create_cont (bool, optional): Create new container. Default is True
+            plugin_path (str, optional): HDF5 vol connector library path.
+                This will enable dfuse (xattr) working directory which is
+                needed to run vol connector for DAOS. Default is None.
+            mount_dir (str, optional): Create specific mount point
+
+        Returns:
+            CmdResult: result of the ior command execution
+
+        """
+        self.processes = processes
+        self.ppn = ppn
+        return self.run_ior_with_pool(
+            create_pool=create_pool, create_cont=create_cont, plugin_path=plugin_path,
+            mount_dir=mount_dir)
```

Comment:
This function just sets two class variables and calls `run_ior_with_pool`, which does a couple other things before calling `run_ior`, which does a couple things before ultimately running the actual ior command. It's this level of obfuscation I'm trying to avoid

Reply:
This function will be removed in one of the previous list of actions